Makefile

```diff
@@ -38,6 +39,48 @@ ifndef IC_PROV_VPC_ID
 endif
 	cd test && TF_ACC=1 go test -v -timeout 120m -count=1
 
+testtarget:
+ifndef TARGET
+	@echo "TARGET for test name must be set for running specific tests"
+	@exit 1
+endif
+
+ifeq (,$(findstring Test, $(TARGET)))
+	@echo "The target is not a test"
+	@exit 1
+endif
+
+ifneq (,$(findstring Acc, $(TARGET)))
```

Comment:
[Desired, maintainability] - so that we don't rely on naming patterns, would it make sense to define these checks on the target tests?

Reply:
Had a discussion of the checks on the target tests and decided that it is creating some duplication especially on specific tests like kafka connect.
We shouldn't be running target tests in the first place imo (tests should run as a whole, and especially now since we are working to make them run automatically during PR phase), but decided to just provide this so we can run tests during development in some way. What we want is that when we run tests, we should just run either `make test` or `make testacc`

Reply:
Yeah there would certainly be a bit of duplication by getting the tests to sort out their own dependencies. It's too bad golang doesn't support annotations because it would have made it easier to mark and check simple test dependencies with something like that. Happy about keeping this as is.